File: trail2/1부터 특정 수까지의 합/sum-from-1-to-a-certain-number.java
import java.util.Scanner;

public class Main {
    public static void sum(int a) {
        int sum = 0;
        for (int i = 1; i <= a; i++) {
            sum += i;
        }
        System.out.print(sum/10);
    }

    public static void main(String[] args) {
        // Please write your code here.
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        sum(n);
    }
}

File: trail2/1부터 특정 수까지의 합/sum-from-1-to-a-certain-number.cpp
#include <iostream>
using namespace std;

void SumNum(int n) {
    int sum = 0;
    for (int i = 1; i <= n; i++) {
        sum += i;
    }
    cout << sum / 10 << endl;
}

int main() {
    // Please write your code here.
    int n;
    cin >> n;
    SumNum(n);

    return 0;
}